audit.sh

```diff
@@ -0,0 +1,406 @@
+#!/bin/bash
+# This script is designed to automate creating and verifying subrepos
+# Every git subrepo clone will be verified with a sha1sum to prove they are identical to the cloned repo
+# Every update/fetch for a sub repo will be presented with an output of the commit differences
+
+# Colours and Formating
+
+REPOURL_RAW="https://raw.githubusercontent.com/rackerlabs/useful-scripts/"
```

Comment:
Doesn't look like this variable is used anywhere?

Reply:
I implemented this in my latest script I just forgot to push the changes here.